Source/Core/AxisAlignedBoundingBox.js

```diff
@@ -191,6 +193,28 @@ define([
         return Intersect.INTERSECTING;
     };
 
+    /**
+     * If the given position is not already within the box, projects the given position onto the box
```

Comment:
I don't think `projection` is the right term here.  This function does clamp the position within the bounds of the box.  `projection` makes me think that the result would be the intersection of the vector for the center of the box to the position with one of the box faces, and this doesn't do that.  I think we just want to update the doc here, this clamping behavior is what we want.